src/providers/info.rs

```diff
@@ -7,6 +7,7 @@ use serde::de::DeserializeOwned;
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 use mockall::automock;
+use crate::utils::json::default_if_null;
```

Comment:
Not that I really mind ... but do we have any convention on the order of imports? (I try to group the `crate` ones and the others .... theoretically you could even try to sort them like it is done in android studio formatting)

Reply:
I think I added this one manually, so that's why it was left somewhere weird. I will move it up.